pyiron/sphinx/base.py

```diff
@@ -1790,7 +1905,8 @@ def get_partial_log(file_content, start_line, end_line):
                 float(line.replace("=", " ").replace(",", " ").split()[1])
                 * HARTREE_TO_EV
                 for line in log_main
-                if line.startswith("eTot(") and not line.startswith("eTot(Val)")
+                if line.startswith("eTot(") and not line.startswith(
+                    "eTot(Val)")
             ]
             energy_zero = 0.5 * (np.array(energy_free) + np.array(energy_int))
```

Comment:
This is not the best way to get these numbers. Better extract them from energy.dat.
For instance, eTot(Val) is the Kohn-Sham energy using the latest (mixed?) density, the Harris-Foulkes energy is more accurate for scfDiag. The energy_zero extrapolation depends on the smearing-scheme order (Methfessel-Paxton or order 1 Fermi-Dirac are different).

Reply:
This is now what @ashtonmv did. I opened [a new issue here](https://github.com/pyiron/pyiron/issues/605)